control_freq: 30.0
xy_tolerance: 0.005
theta_tolerance: 0.01
backward_tolerance: 0.0

deviate_distance: 0.10
deviate_angle: 0.2
# Kd: 2
# Kt: 5
Kd: 2
Kt: 2.5

latch_xy: true
linear_compensate: true
overshoot_compensate: true

max_linear_velocity: 0.8
v_threshold: 0.05
velocity_acc: 0.5
velocity_deceleration: 0.3
v_recovery: 0.05

max_angular_velocity: 1.2
w_threshold: 0.1
angular_velocity_acc: 2.0
angular_velocity_deceleration: 1.0
w_recovery: 0.1

tolerance_input: [[6,0.05,0.2] ]

# max_linear_velocity: 0.3
# v_threshold: 0.05
# velocity_acc: 0.25
# velocity_deceleration: 0.1
# v_recovery: 0.05

# max_angular_velocity: 1.2
# w_threshold: 0.1
# angular_velocity_acc: 1.0
# angular_velocity_deceleration: 0.8
# w_recovery: 0.1

